README.md

```diff
@@ -1745,6 +1751,8 @@ hz.getQueue('my-distributed-queue').then(function (q) {
 });
 ```
 
+Hazelcast Queue uses `ItemListener` to listen to events that occur when items are added to and removed from the Queue. See the [Item Listener section](#7523-item-listener) for information on how to create an item listener object and register it.
```

Comment:
Hazelcast Queue uses `ItemListener` to listen to the events that occur when the items are added to or removed from the Queue.